<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>BootLoaderWidget</class>
  <widget class="QWidget" name="BootLoaderWidget" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>611</width>
        <height>517</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Form1</string>
    </property>
    <layout class="QGridLayout" >
      <item row="3" column="2" >
        <spacer name="spacer5_2" >
          <property name="sizeHint" >
            <size>
              <width>40</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Horizontal</enum>
          </property>
        </spacer>
      </item>
      <item row="0" column="1" >
        <spacer name="spacer17" >
          <property name="sizeHint" >
            <size>
              <width>20</width>
              <height>16</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
      <item row="3" column="0" >
        <spacer name="spacer5" >
          <property name="sizeHint" >
            <size>
              <width>40</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Horizontal</enum>
          </property>
        </spacer>
      </item>
      <item rowspan="1" row="1" column="0" colspan="3" >
        <widget class="QLabel" name="textLabel1" >
          <property name="lineWidth" >
            <number>1</number>
          </property>
          <property name="margin" >
            <number>35</number>
          </property>
          <property name="text" >
            <string>&lt;b>Do you want to write boot loader (GRUB) to your hard disk's master boot record?&lt;/b>

&lt;p>You need a boot loader to boot your Pardus system.&lt;/p>

&lt;p>If you don't install a boot loader or choose to install to root partition, you will have to configure your other operating system's loader  (if you have one installed) to start 
your newly installed Pardus.&lt;/p></string>
          </property>
          <property name="alignment" >
            <set>Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap" >
            <bool>true</bool>
          </property>
        </widget>
      </item>
      <item row="2" column="1" >
        <spacer name="spacer5_3" >
          <property name="sizeHint" >
            <size>
              <width>31</width>
              <height>16</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
      <item row="4" column="1" >
        <spacer name="spacer3" >
          <property name="sizeHint" >
            <size>
              <width>20</width>
              <height>16</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
      <item row="3" column="1" >
        <widget class="Q3ButtonGroup" name="buttonGroup" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>5</hsizetype>
              <vsizetype>4</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="title" >
            <string>Where do you want to install the bootloader?</string>
          </property>
          <layout class="QGridLayout" >
            <item row="0" column="0" >
              <widget class="QRadioButton" name="installFirstMBR" >
                <property name="text" >
                  <string>Install to first bootable disk (recommended)</string>
                </property>
              </widget>
            </item>
            <item row="1" column="0" >
              <widget class="QRadioButton" name="installPart" >
                <property name="text" >
                  <string>Install to Pardus installed partition</string>
                </property>
              </widget>
            </item>
            <item row="3" column="0" >
              <widget class="QRadioButton" name="installMBR" >
                <property name="text" >
                  <string>Install to selected disk belove</string>
                </property>
              </widget>
            </item>
            <item row="4" column="0" >
              <layout class="QVBoxLayout" >
                <item>
                  <widget class="QLabel" name="select_disk_label" >
                    <property name="text" >
                      <string>Select the disk you to install bootloader:</string>
                    </property>
                    <property name="wordWrap" >
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="Q3ListBox" name="device_list" >
                    <property name="enabled" >
                      <bool>false</bool>
                    </property>
                    <property name="frameShadow" >
                      <enum>QFrame::Plain</enum>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
            <item row="5" column="0" >
              <widget class="QRadioButton" name="noInstall" >
                <property name="text" >
                  <string>Don't install bootloader</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <tabstops>
    <tabstop>installFirstMBR</tabstop>
    <tabstop>installPart</tabstop>
    <tabstop>installMBR</tabstop>
    <tabstop>device_list</tabstop>
    <tabstop>noInstall</tabstop>
  </tabstops>
  <connections>
    <connection>
      <sender>installMBR</sender>
      <signal>toggled(bool)</signal>
      <receiver>device_list</receiver>
      <slot>setEnabled(bool)</slot>
    </connection>
  </connections>
</ui>
